app/src/constants.ts

```diff
@@ -0,0 +1,36 @@
+type EnhancedWindow = typeof window & {
```

Comment:
maybe you can put this in the services folder ? it concerns external services

Reply:
I could have, but PUBLIC_URL does not concern external services...
Also, I like when variables injected from the environment are defined at only one place, so I'd rather not put it in services/api/urls.ts.
do you prefer services/constants.ts ?